example/README.md

```diff
@@ -36,14 +28,21 @@
 
     $ ./monorail_rack
 
-  Copy local basic static files to common directory:
-
-    $ cp ~/<static files directory>/* ~/<repos directory>/RackHD/on-http/static/http/common/
-
   Now ssh into the monorail server:
 
     $ vagrant ssh dev
 
+  Build the discovery images from the monorail server (or any debian-based system):
+
+    $ sudo apt-get install -y ansible
+    $ git clone https://github.com/rackhd/on-imagebuilder.git
+    $ cd on-imagebuilder
+    $ sudo ansible-playbook -i hosts all.yml
+
+  Place the discovery images into HTTP server's static file directory:
+
+    $ mv /tmp/on-imagebuilder/builds/* ~/<repos directory>/RackHD/on-http/static/http/common/
```

Comment:
if this is build inside the VM then we actually want to move them to the on-http repo through /home/vagrant/src/on-http/static/http/common/